### ISSUE TITLE
Unity 2018.3: Macro Redefinition and UnityWebRequest.Send() is obsolete warnings

### ISSUE THREAD

**Issue by SoftwareGuy** [Xiexe/Xiexes-Unity-Shaders#6]
Greetings,

Just a few warnings that popped up after importing 2.0.4 after cleaning 1.6.1 out of my game project. Are these intended or are they harmless? I'm using Unity 2018.3.9f1.

```
Shader warning in 'Xiexe/Toon2.0/XSToon2.0': 'UNITY_PASS_FORWARDBASE' : macro redefinition at line 115 (on d3d11)

Compiling Vertex program with UNITY_PASS_FORWARDBASE DIRECTIONAL
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_FULL_HDR

Shader warning in 'Xiexe/Toon2.0/XSToon2.0': 'UNITY_PASS_FORWARDADD' : macro redefinition at line 139 (on d3d11)

Compiling Vertex program with UNITY_PASS_FORWARDADD POINT
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_FULL_HDR

Shader warning in 'Xiexe/Toon2.0/XSToon2.0': 'UNITY_PASS_SHADOWCASTER' : macro redefinition at line 159 (on d3d11)

Compiling Vertex program with UNITY_PASS_SHADOWCASTER SHADOWS_DEPTH
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_FULL_HDR
```
There's also this obsolete warning message, but it's probably not going to an issue until later versions:
```
Assets\Shaders\Xiexe's Unity Shaders\Editor\XSUpdater.cs(120,9): warning CS0618: 'UnityWebRequest.Send()' is obsolete: 'Use SendWebRequest.  It returns a UnityWebRequestAsyncOperation which contains a reference to the WebRequest object.'
```

Thanks for making these great toon/anime shaders. I'll definitely be using them in my project and will credit you.

Cheers!

**Comment by Xiexe** [comment]
The UnityWebRequest thing will be fixed in 2.0.5 when I release that soon with a few other fixes. 

The macro redefinitions should be harmless, I think 2018 added them to the #pragma fwdbase/fwdadd macros, but they aren't there in 2017. So I assume that's a new thing - I believe it should be harmless.

**Comment by ScruffyRules** [comment]
UnityWebRequest fixed in [2.0.5](https://github.com/Xiexe/Xiexes-Unity-Shaders/releases/tag/2.0.5)
